dsp-schema-repo/src/main/scala/dsp/schema/repo/SchemaRepoTest.scala

```diff
@@ -0,0 +1,24 @@
+package dsp.schema.repo
+
+import zio._
+import dsp.schema.domain.SchemaDomain.{UserID, UserProfile}
+
+case class SchemaRepoTest() extends SchemaRepo {
+  private var map: Map[UserID, UserProfile] = Map()
```

Comment:
We agreed on never using `var`. Is there a good reason to use it here instead of `val`? If so, what is the reason?

Reply:
to every rule there are exceptions :-) This is the simplest way of implementing a key-value store. Since it is only for testing, it should be fine. I "stole" the code from John workshop.